### ISSUE TITLE
[Bug]: False positive for LC0086 on OptionCaption = 'None'

### ISSUE THREAD

**Issue by MODUSCarstenScholling** [ALCops/Analyzers#376]
### Analyzer

LinterCop

### Issue type

False positive (diagnostic fires when it shouldn't)

### Rule ID

LC0086

### Description

LC0086 fires for single OptionCaption containing one of the Style words.

Avoid using the string literal 'None' for page styling. Use the PageStyle datatype instead (PageStyle::None).

### AL code to reproduce

```al
	field(35583; "Option"; Option)
	{
		Caption = 'Option', Locked = true;
		OptionMembers = None;
		OptionCaption = 'None', Locked = true;
	}
```

**Comment by MODUSCarstenScholling** [comment]
@Arthurvdv I hope it is OK that I fix this on-the-fly in the 1.0.0 branch with my existing PR?

**Comment by Arthurvdv** [comment]
Would you mind opening a separate PR for this? I tend to squash-merge PRs, so I prefer to keep unrelated changes in separate PRs rather than combining multiple issues into one.

**Comment by MODUSCarstenScholling** [comment]
Of course. But still into release/v1.0.0?

**Comment by Arthurvdv** [comment]
Jep!
